modules/cluster/outputs.tf

```diff
@@ -1,6 +1,8 @@
 output "installer_dashboard_url" {
   description = "The URL to access the installer dashboard."
-  value       = "https://${google_compute_instance.primary[0].network_interface[0].access_config[0].nat_ip}:8800"
+  value = length(google_compute_instance.primary) > 0 ? (
+    "https://${google_compute_instance.primary[0].network_interface[0].access_config[0].nat_ip}:8800"
+  ) : ""
```

Comment:
we might want to just add a comment around what the case for this is, cuz coming in cold to this, I can see someone being like "you'll never have no primaries"

Reply:
I think I'd actually like to see the output be a list of all the deployed primary instances with :8800 on the end
Like, go to any of these, instead of picking one for them.

We should circle back on the convo to change how we hit the dashboard, too.

Reply:
I'm on the other side of the fence on this one, I would rather have a single endpoint for the customer to go thru, but we could maybe have it behind an LB or as a dns entry pointing at all of them.

I agree we should circle back to discussing this in a broader context

Reply:
Resolved by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/45/commits/34f7e9da1e20df1386c2f70588c265e200e029cf.